services/_internal/maven-EGI/workflowEGI/src/main/java/it/infn/ct/configEGI.json

```diff
@@ -0,0 +1,8 @@
+{
+	"endpoint": "https://carach5.ics.muni.cz:11443",
```

Comment:
It looks like you already have this URL hard coded in multiple files. Why don't you read it from the config file instead? It could make future code modifications much easier